src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const employeesNodeList = document.querySelectorAll('body > ul > li');
+const ulNode = document.querySelector('body > ul');
+const convertToObjects = (nodeList) => {
+  const arrOfEmployees = [];
+
+  for (const employee of Array.from(nodeList)) {
+    arrOfEmployees.push({
+      name: employee.textContent.trim(),
+      salary: employee.getAttribute('data-salary'),
+      age: employee.getAttribute('data-age'),
+      position: employee.getAttribute('data-position'),
+    });
+  }
+
+  return arrOfEmployees;
+};
+const sortArrBySalary = (arr) => {
+  const convertStrToNumber = (str) => Number(str.replaceAll(',', '')
+    .replaceAll('$', ''));
```

Comment:
```suggestion
  const convertStrToNumber = (str) => Number(str.slice(1).replaceAll(',', ''));
```
```suggestion
  const convertStrToNumber = (str) => Number(str.replaceAll(',', '')
    .replaceAll('$', ''));
```
`$` is always in the very beginning

Reply:
ok, but what is the purpose of this change? this will make code harder to read